src/shogun/kernel/GaussianKernel.h

```diff
@@ -189,7 +189,7 @@ class CGaussianKernel: public CDotKernel
 
 	protected:
 		/** width */
-		float64_t width;
+		float64_t m_log_width;
```

Comment:
It seems that `the integration tests` are generated by some scripts.
I have to modify the scripts to pass the test.
Could you tell me the path the the scripts?